src/main/webapp/app/entities/modeling-exercise/modeling-exercise.module.ts

```diff
@@ -37,18 +36,11 @@ const ENTITY_STATES = [...modelingExerciseRoute, ...modelingExercisePopupRoute];
     declarations: [
         ModelingExerciseComponent,
         ModelingExerciseDetailComponent,
-        ModelingExerciseDialogComponent,
-        ModelingExerciseDeleteDialogComponent,
-        ModelingExercisePopupComponent,
-        ModelingExerciseDeletePopupComponent,
-    ],
-    entryComponents: [
-        ModelingExerciseComponent,
-        ModelingExerciseDialogComponent,
-        ModelingExercisePopupComponent,
+        ModelingExerciseUpdateComponent,
         ModelingExerciseDeleteDialogComponent,
         ModelingExerciseDeletePopupComponent,
     ],
+    entryComponents: [ModelingExerciseComponent, ModelingExerciseDeleteDialogComponent, ModelingExerciseDeletePopupComponent],
```

Comment:
Not from this PR, but while we are at it we could remove the `ModelingExerciseComponent` here.

Reply:
Sure, better to do it earlier